gemini/tests/set3/selects.js

```diff
@@ -11,9 +11,9 @@ gemini.suite('selects', child => {
   child
     .setUrl('/selects')
     .before((actions, find) => {
-      actions.waitForElementToShow('.form-block', WAIT_TIME);
+      actions.waitForElementToShow('.content-area', WAIT_TIME);
```

Comment:
Any reason not to make the reference image smaller and more targeted?

Reply:
This required adding a new selector to the page in another test. I'm not sure why it renders so large, must be part of Gemini?